spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper_acceptance'
+
+context 'trusted external data' do
```

Comment:
I think acceptance tests should be structured as "We're testing this feature." In this case, the feature's "trusted external data". "trusted external command" is an implementation detail.